activiti-cloud-query-service/activiti-cloud-starter-query/src/main/resources/config/query/liquibase/changelog/06-alter.pg.schema.7.1.0-m9.sql

```diff
@@ -0,0 +1,4 @@
+alter table process_instance
+    add completed_date timestamp;
```

Comment:
Please check, but I think the syntax is wrong for Postgres. It should be `add column`.

Reply:
Updated the postgres db command.  Thanks.